### PR TITLE
Maven to Gradle Migration

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
## Change list

Maven to Gradle Migration
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

We will be using Gradle to build instead of Maven from our next release.Details are here:
https://github.com/appium/java-client/issues/214

### CODE REVIEW

**Comment by SrinivasanTarget** [comment]
@TikhomirovSergey FYI :)

**Comment by SrinivasanTarget** [comment]
@bootstraponline You can also join this PR :)

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+}
+
+apply plugin: "org.owasp.dependencycheck"
+
+configurations {
+    ecj
+}
+
+dependencies {
+    ecj 'org.eclipse.jdt.core.compiler:ecj:4.4'
```
Comment (TikhomirovSergey) [inline build.gradle]: How about v4.5.1?

File: pom.xml [hunk 2]
```diff
@@ -1,362 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (TikhomirovSergey) [inline pom.xml]: I would like to keep this file for sometime. 
Could you just rename this file to `pom_.xml`

File: build.gradle (hunk 1 above)
Comment (SrinivasanTarget) [inline build.gradle]: Sure, i will update it.

File: pom.xml (hunk 2 above)
Comment (SrinivasanTarget) [inline pom.xml]: @TikhomirovSergey Does it create any issues if we keep this file just as it is as ```pom.xml``` for sometime?

Comment (TikhomirovSergey) [inline pom.xml]: @SrinivasanTarget 
I was trying it, There was no problem

Comment (SrinivasanTarget) [inline pom.xml]: Cool i wont delete pom.xml. Is that fine?

**Comment by TikhomirovSergey** [comment]
@SrinivasanTarget 
There are problems with the importing/test running on IDE. I use IntellijIDEA 15.x and 2016.3/4. Please try to import the project. 

PS: Don't forget to rename `pom.xml` to `pom.xml_` for the history. This file may be helpful for some time

**Comment by SrinivasanTarget** [comment]
Ok @TikhomirovSergey I will look into it today

**Pull request merged**